notebooks/non_ioos_sources.ipynb

```diff
@@ -36,8 +36,8 @@
     "glider_grab = GliderDataFetcher(\"http://www.ifremer.fr/erddap\")\n",
     "glider_grab.fetcher.dataset_id = \"OceanGlidersGDACTrajectories\"\n",
     "glider_grab.query(-90, 90, -180, 180, \"2015-09-20\", \"2015-09-27\")\n",
-    "df = glider_grab.to_pandas()\n",
-    "df.head()"
+    "df_ifr = glider_grab.to_pandas()\n",
+    "df_ifr.head()"
```

Comment:
I would avoid naming the dataframes in the examples just to make them more copy-n-pastable from multiple notebooks.

Reply:
Good point. Fixed with commit a9190a7